src/sample/CancelAccessTokenSample.js

```diff
@@ -3,12 +3,13 @@
  *
  * @param {Member} grantor - grantor member
  * @param {string} tokenId - id of the token to cancel
+ * @param {bool} nonBlocking - flag to create a non-blocking request
  * @return {Object} result - result of the token operation
  */
-export default async (grantor, tokenId) => {
+export default async (grantor, tokenId, nonBlocking) => {
```

Comment:
these should be blocking not non-blocking right?

Reply:
right, I got mixed up XD